app/lib/saml.server.ts

```diff
@@ -67,13 +67,13 @@ const sp = samlify.ServiceProvider({
   assertionConsumerService: [
     {
       Binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST',
-      Location: process.env.HOSTNAME + '/login/callback',
+      Location: '/login/callback',
```

Comment:
Is this allowed (relative URL)?  I suspect we need an absolute, and what you had before was likely correct.

Reply:
Yeah I was messing around to see if it makes a difference and everything still worked, I think this hints at a problem with what I wrote. I will change this back though.